src/index.js

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+import ReactDOM from "react-dom/client";
```

Comment:
I noticed that two different approaches for rendering the React app are being used in the project:

The first approach uses createRoot from react-dom/client with StrictMode, which is the recommended practice for React 18+ as it supports features like Concurrent Rendering and enforces best practices during development.
The second approach also uses ReactDOM.createRoot but omits StrictMode, simplifying the code but not fully leveraging React's modern features.

Please ensure the initial structure of the project remains unchanged. You are welcome to add extra tweaks for home assignments or pursue stretch goals, but the original setup should not be modified.

Reply:
Thanks for the feedback! I added React.StrictMode around <App /> to follow the recommended approach for React 18+.